tools/wdio/runner.mjs

```diff
@@ -30,9 +30,17 @@ if (['trace', 'debug', 'info'].indexOf(wdioConfig.logLevel) > -1) {
 // Clear the JIL CLI params before starting wdio so they are not passed to worker processes
 process.argv.splice(2)
 
+const jsonReplacer = (_, value) => {
+  if (value instanceof RegExp) {
+    return value.toString()
+  }
```

Comment:
This won't work to get a RegExp object back when deserializing; I've tried in JIL before (after spending too much 💀 time).
![image](https://user-images.githubusercontent.com/28714891/236909696-2f69444d-e854-44a3-bd30-7fc3e0d7e40a.png)

If can't use serialize-anything, you can take from the example in https://github.com/newrelic/newrelic-browser-agent/blob/ba7d17cd3cf3b0a665d8e1a4e1b1bba17d19e1df/tools/testing-server/utils/url.js#L30-L36 , but still needs to de-serialize it with replace (see init-transform.js, L27-30).

Reply:
This works for this purpose because of how WDIO parses the configuration. The point of the `replacer` is to simply convert the `RegExp` to a string representation which WDIO supports. We do not ever parse the config after starting WDIO, WDIO does all that so we cannot use `serialize-anything`.